resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -345,8 +345,11 @@ class KubernetesSuite extends SparkFunSuite
                 }
                 // Delete the pod to simulate cluster scale down/migration.
                 // This will allow the pod to remain up for the grace period
+                // We set an intentionally long grace period to test that Spark
+                // exits once the blocks are done migrating and doesn't wait for the
+                // entire grace period if it does not need to.
                 kubernetesTestComponents.kubernetesClient.pods()
-                  .withName(name).delete()
+                  .withName(name).withGracePeriod(8640000).delete()
```

Comment:
Although this is a test case, `8640000` looks like a magic number. Is there a reason to choose this?

Reply:
Is this due to https://github.com/apache/spark/pull/31249/files#diff-1c2fb0706df8a5c08f357f108f907239a945d3b857c3d2a774c59fc083b57416R130 ?

Reply:
This is just a really long value, we can pick any large number here.

Reply:
In that case, can we use `Int.MaxValue` simply?

Reply:
Sounds good to me.